File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/CongViecTapTinDinhKem.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CongViecTapTinDinhKem } from 'src/app/shared/CongViecTapTinDinhKem.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CongViecTapTinDinhKemService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CongViecTapTinDinhKem";
    }
}

File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/CongViecTapTinDinhKemService.cs
namespace Service.Implement
{
    public class CongViecTapTinDinhKemService : BaseService<CongViecTapTinDinhKem, ICongViecTapTinDinhKemRepository>
    , ICongViecTapTinDinhKemService
    {
    private readonly ICongViecTapTinDinhKemRepository _CongViecTapTinDinhKemRepository;
    public CongViecTapTinDinhKemService(ICongViecTapTinDinhKemRepository CongViecTapTinDinhKemRepository) : base(CongViecTapTinDinhKemRepository)
    {
    _CongViecTapTinDinhKemRepository = CongViecTapTinDinhKemRepository;
    }
    }
    }

